lib/src/lightclient.rs

```diff
@@ -589,19 +600,25 @@ impl LightClient {
             let balance = self.wallet.tbalance(Some(taddress.clone())).await;
 
             t_addresses.push(object! {
-                "address" => taddress,
-                "balance" => balance,
+                "address"                     => taddress,
+                "balance"                     => balance,
             });
         }
 
+        // TODO:  Think about exposing an interface that references the Orchard Pool.
         object! {
-            "zbalance"           => self.wallet.zbalance(None).await,
-            "verified_zbalance"  => self.wallet.verified_zbalance(None).await,
-            "spendable_zbalance" => self.wallet.spendable_zbalance(None).await,
-            "unverified_zbalance"   => self.wallet.unverified_zbalance(None).await,
-            "tbalance"           => self.wallet.tbalance(None).await,
-            "z_addresses"        => z_addresses,
-            "t_addresses"        => t_addresses,
+            "sapling_balance"                 => self.wallet.sapling_balance(None).await,
```

Comment:
Almost repetitive enough for a declarative macro?

Reply:
Hmm.. That's a good thought. Issued.